bin/fx-runner

```diff
@@ -1,34 +1,39 @@
 #!/usr/bin/env node
 
 var VERSION = require("../package.json").version;
-var program = require("commander");
+var { program } = require("commander");
 var run = require("../lib/run");
 
 program
   .version(VERSION)
+
+program
+  .command("start")
+  .description("Start Firefox")
   .option("-b, --binary <path>", "Path of Firefox binary to use.")
   .option("--binary-args <CMDARGS>", "Pass additional arguments into Firefox.")
   .option("-p, --profile <path>", "Path or name of Firefox profile to use.")
   .option("-v, --verbose", "More verbose logging to stdout.")
   .option("--new-instance", "Use a new instance")
   .option("--no-remote", "Do not allow remote calls")
   .option("--foreground", "Bring Firefox to the foreground")
-  .option("-l, --listen <port>", "Start the debugger server on a specific port.")
+  .option("-l, --listen <port>", "Start the debugger server on a specific port.", parseInt)
+  .action(function(options) {
+    const parsedOptions = {
+      binary: options.binary || process.env.JPM_FIREFOX_BINARY || "firefox",
+      profile: options.profile,
+      "new-instance": !!options.newInstance ? true : false,
+      "foreground": !!options.foreground ? true : false,
+      "no-remote": !options.remote ? true : false,
+      "binary-args": options.binaryArgs || "",
+      "listen": options.listen || 6000
+    };
 
-program
-  .command("start")
-  .description("Start Firefox")
-  .action(function() {
-    console.log(Object.keys(program))
```

Comment:
The test harness relied on `program` being outputted to the user. Commander no longer applies the flags to `program` and they no longer appear as `--flags` anymore, so the test had to be updated.

I made two changes:

- the user no longer sees this noise, it's hidden behind a fx-runner-specific ENV
- instead of looking for random flags, it matches against the whole `parsedOptions` object. This ensures that the additional logic in this object can also be tested.